mandatory/2-function-creation.js

```diff
@@ -6,7 +6,21 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof(num) == 'string') {
+    return false
+  } else {
+  if (num <= 100) {
+    if (num % 2 === 0) {
+      return true;
+    } else {
+      return false;
+    }
+  }  else {
+      return false;
+  }
+}
+}
```

Comment:
just as an idea: i think it is possible to use here "else if" and it would make the code more readable. two "else"s at the moment look like repeating